Detectors/CTP/workflowLumi/src/lumiWorkflowSpec.cxx

```diff
@@ -0,0 +1,42 @@
+// Copyright 2019-2020 CERN and copyright holders of ALICE O2.
```

Comment:
please rename this file to lumiWorkflow.cxx, the "Spec" is for devices, while this one defines the workflow.
also, you don't need corresponding header file.